packages/cxl-ui/src/components/cxl-featured-course-card.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { LitElement, html } from 'lit';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { customElement, property } from 'lit/decorators.js';
+import '@conversionxl/cxl-lumo-styles';
+import cxlFeaturedCourseCardStyles from '../styles/cxl-featured-course-card-css.js';
+
+@customElement('cxl-featured-course-card')
+export class CXLCourseCardElement extends LitElement {
+  static get styles() {
+    return [cxlFeaturedCourseCardStyles];
+  }
+
+  separator = html`<span> | </span>`;
+
+  @property({ type: String }) id = '';
+
+  @property({ type: String }) featuredCourse = 'Featured course';
```

Comment:
I think this property could have different, more meaningful name but nothing comes to my mind...

Reply:
I renamed this to 'theme'. But probably it will be changed again when card components will be refactored.